jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -272,7 +272,15 @@ public void saveSearchFilterSettings(final String filterName,
         settings.setTableName(filterName);
         settings.setTableDescription(filterName);
         filterSettingsManager.saveFilterIntoPreferences(settings,
-                                                        state -> callback.accept(state ? null : Constants.INSTANCE.FilterWithSameNameAlreadyExists()));
+                                                        state -> {
+                                                            if (state) {
+                                                                getListView().getListGrid().getGridPreferencesStore().setPreferenceKey(settings.getKey());
```

Comment:
@nmirasch do we still need to do this? should be set already right?

Reply:
@cristianonicolai yes, this is the moment to set the PreferenceKey used to store the grid preferences, even any of next preference changes won't be persisted.